src/content/wallet.json

```
@@ -160,6 +160,57 @@
       }
     ]
   },
+  {
+    "component": "Wallet/FeatureCards",
+    "title": "<b>More features beyond</b> multisigs",
+    "text": "Safe is more than just secure asset storage. It's your entry point to onchain activities.",
+    "items": [
+      {
+        "isNew": true,
+        "fullWidth": true,
+        "caption": "Trade seamlessly",
+        "title": "Swap like a pro",
+        "text": "with MEV protection and security, ensuring your intent-based transactions will not fail.",
+        "image": {
+          "src": "/images/Wallet/Features/trade.png",
+          "alt": "Token swap modal"
+        }
+      },
+      {
+        "caption": "Optimise",
```

I think we adhere to American English.

```suggestion
        "caption": "Optimize",
```